DOCS/man/options.rst

```diff
@@ -1200,7 +1200,7 @@ Video
     .. note::
 
         Use the ``Ctrl+h`` shortcut to toggle hardware decoding at runtime. It
-        toggles this option between ``auto`` and ``no``.
+        toggles this option between ``auto-safe`` and ``no``.
```

Comment:
Wouldn't it be better to actually toggle on user settings between `hwdec` and `no`? 

It is always strange to start with some hwdec and after toggling ctrl+h end up with a different one.

Reply:
As I discussed elsewhere, this would require a change to how configuration works - You can't toggle between `hwdec` and `no` because ctrl+h rewrites `hwdec`, so we'd need to add a new config option for "the hwdec list to use with ctrl+h". 

Which is doable, but I'm not bundling it in here.

Reply:
I don't think you need 2nd config option for that. ctrl+h could simply remember current hwdec when toggling and restore it on next toggle. I haven't look into details, but with one more state variable it should be doable. But fair, it can be done in different PR in the future.